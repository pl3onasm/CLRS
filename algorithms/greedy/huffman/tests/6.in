I do not think that I know what I do not know