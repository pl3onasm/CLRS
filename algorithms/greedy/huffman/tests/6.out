CODEWORD TABLE

char      freq       code
   a         2       0000
   i         1       00010
   d         2       00011
   n         5       001
            11       01
   t         6       100
   I         3       1010
   k         3       1011
   o         6       110
   w         3       1110
   h         3       1111


HUFFMAN TREE

-0-0-0-0 [2] 'a'
-0-0-0 [5]
-0-0-0-1-0 [1] 'i'
-0-0-0-1 [3]
-0-0-0-1-1 [2] 'd'
-0-0 [10]
-0-0-1 [5] 'n'
-0 [21]
-0-1 [11] ' '
root [45]
-1-0-0 [6] 't'
-1-0 [12]
-1-0-1-0 [3] 'I'
-1-0-1 [6]
-1-0-1-1 [3] 'k'
-1 [24]
-1-1-0 [6] 'o'
-1-1 [12]
-1-1-1-0 [3] 'w'
-1-1-1 [6]
-1-1-1-1 [3] 'h'

